### ISSUE TITLE
Add possibility to disable calendar notifications in Web interface

### ISSUE THREAD

**Issue by ogasser** [nextcloud/server#42659]
After upgrading to Nextcloud 28 I get a notification in the Nextcloud Web interface for all calendar events with notifications. As I use Thunderbird instead of the Web interface for interacting with the Nextcloud calendars, I find this quite annoying. It would be great if the possibility to disable calendar event notifications could be added to the personal notification settings:

![image](https://github.com/nextcloud/server/assets/2095178/3e7fd14b-ce88-48cd-8c79-119b11564bf0)

**Comment by Linkinsoldier** [comment]
I agree. Also me and other people in the forum (https://help.nextcloud.com/t/ungewollte-benachrichtigung-fur-termine/178137) are very unhappy, that we are now pushed every calender event via the nextcloud app (to our android phones). We now see calender event notifications twice (once in the calender app and once in nextcloud). This did never happen in the last 10 years with NC27 or older. 

Can we please make the user able to change this? 

PS: WAF is really getting down right now because of the multiple notifications ;)

**Comment by ChristophWurst** [comment]
This was added intentionally: https://github.com/nextcloud/server/issues/39862. You can opt-out of the feature.

**Comment by ogasser** [comment]
@ChristophWurst how exactly can we opt-out of the feature?

**Comment by ChristophWurst** [comment]
``php occ config:app:set dav sendEventRemindersPush--value=no``

https://docs.nextcloud.com/server/stable/admin_manual/configuration_server/occ_command.html

**Comment by ogasser** [comment]
Thanks, that works! :+1:

I still think it would be good to expose this as a per-user setting accessible through the web interface. @jancborchardt thoughts?

**Comment by ogasser** [comment]
I applied the setting, but I still get calendar notifications in the browser. Any idea why @ChristophWurst ?

**Comment by Linkinsoldier** [comment]
I hope this can be disables per web settings in the feature. I don't have access to occ and therefore can not disable it ...!

**Comment by Martin511** [comment]
@ChristophWurst 
Pleeeeaaaase make this accessible as a per-user setting through the web interface as many of us do not have occ access.
It is annoying.
Thanks

**Comment by ChristophWurst** [comment]
I don't have time to look into this myself but I'm happy to guide anyone who would like to contribute a fix.

**Comment by andre5tz** [comment]
I would be very happy if the settings could be made for each individual calendar.
As soon as I have access to other calendars, the notifications will also be sent to me, even though the view is disabled.

**Comment by joshtrichards** [comment]
This is already implemented unless I'm missing something:

https://github.com/nextcloud/server/blob/1a6440c353d3c797140215e4ab885544f1ac3741/apps/dav/src/views/CalDavSettings.vue#L67-L73

It's a toggle visible under *Administration settings->Groupware->Calendar*

**Comment by Linkinsoldier** [comment]
well i guess the problem is, that the Groupware seems to be only available to those who can also access occ (i guess). I would have expected the setting to be in the notifications settings. 

I can not load the groupeware side:
![Image](https://github.com/user-attachments/assets/bab28930-db46-47d4-b323-035aceffd759)

**Comment by ChristophWurst** [comment]
@Linkinsoldier thanks for the analysis. The settings section is for admins, and it won't show if there are no frontend apps that need a caldav backend. You can find more details at https://github.com/nextcloud/server/blob/42c021fda262685125f71e2492336fdc503d5b68/apps/dav/lib/Settings/CalDAVSettings.php#L49-L55.

**Comment by Linkinsoldier** [comment]
I dont get it. The Calender itself has the option to either send this notifications or not. there is the option to switch this off via:
php occ config:app:set dav sendEventRemindersPush --value=no

and i understood the post above to mean, that there is no necessarity to add this option in the notifications settings, because it is already in the Groupware settings. I only stated, that the groupware options are not available to all user (same as the php occ config). I therefor would suggest to have an option in notification settings to make it available for the user.

**Comment by joshtrichards** [comment]
@Linkinsoldier Look at the title of the Issue. That's what this enhancement Issue is about. It's still open = so not implemented on a per-user basis.

Admins, however, can manage it globally via both occ and the admin web UI.
